### DISCUSSION TITLE
Getting git push to update everything in my copied directory

### DISCUSSION THREAD

**Discussion by ark561** [quarto-dev/quarto-cli#5170] [Q&A] [answered]
Hello again!

I feel like I am ever so close to getting things to work. 

First, I feel like I am not getting the .nojekyll file to create properly. When I am in my copied directory, I type `touch .nojekyll`, but nothing happens. This may be the core issue, I have.

I have used the `quarto render` command and then `git push`, nothing on my directory changes. I don't get the `docs` folder uploaded, so I must be missing something.

If more information is needed, please let me know. My apologies if I am being unclear; I am working through the guides on quarto, but am likely missing some of the prerequisite knowledge.

Thanks in advance for helping this old dog to learn some new tricks!

**Comment by rgaiacs** [comment, marked as answer]
Dear @ark561,

from the shell/bash command line, you can run

```
ls -a
```

and you should see a file `.nojekyll`.

You **must** run

```
git add -f docs
```

follow by

```
git commit -m 'Add HTML built with quarto'
```

and

```
git push origin main
```

Let me know if you need more details.

**Reply by ark561** [reply]
This worked!

Thank you so much!!
